odl/solvers/advanced/proximal_operators.py

```diff
@@ -31,7 +31,7 @@
 from builtins import super
 
 import numpy as np
-import scipy.special
+import scipy
```

Comment:
The failure is due to `AttributeError: module 'scipy' has no attribute 'special'`. But the tests run on my machine... and in all places I do `scipy.special.lambertw`